Image plugin installation does not preserve the file's hierarchy

This issue was first discovered by @rosaliepl : https://www.sofa-framework.org/community/forum/topic/error-with-cimg-plugin-with-dependancy-to-skeleton-h-does-not-compile/#post-8397

When installing SOFA's image plugin (ex.: make install), the image plugin installation does not preserve the file's hierarchy which will failed the compilation of any external project that includes the SOFACImg.h file.

`/.../sofa/build/install/include/CImg/CImg.h:9341:21: fatal error: plugins/skeleton.h: No such file or directory`

If we look in the installation folder (sofa/build/install), all the required headers are in include/CImg instead of include/CImg/plugins which will then failed the compilation.

To reproduce the bug

1. Compile and install SOFA with the image plugin activated
2. Create an external project with the image plugin as a requirement, and add the following line in any of the source file: `#include <CImg/SOFACImg.h>`

--- a/applications/plugins/image/extlibs/CImg/CMakeLists.txt
+++ b/applications/plugins/image/extlibs/CImg/CMakeLists.txt
@@ -2,27 +2,14 @@ cmake_minimum_required(VERSION 3.1)
 
 project(CImg)
 
-install(FILES 
-    CImg.h
-    SOFACImg.h
-    plugins/add_fileformat.h
-    plugins/chlpca.h
-    plugins/draw_gradient.h
-    plugins/ipl_alt.h
-    plugins/jpeg_buffer.h
-    plugins/matlab.h
-    plugins/patchmatch.h
-    plugins/tiff_stream.h
-    plugins/vrml.h
-    plugins/bayer.h
-    plugins/cvMat.h
-    plugins/inpaint.h
-    plugins/ipl.h
-    plugins/loop_macros.h
-    plugins/nlmeans.h
-    plugins/skeleton.h
-    plugins/tinymatwriter.h
-    plugins/vtk.h
+install(FILES
+        CImg.h
+        SOFACImg.h
+        DESTINATION include/CImg
+        COMPONENT CImg_headers)
+
+install(DIRECTORY
+        plugins
         DESTINATION include/CImg
         COMPONENT CImg_headers)
 

